process_bigraph/composite.py

```diff
@@ -559,8 +518,8 @@ def process_update(
             process,
             states,
             interval,
-            ports_key=None,
-    ):
+            ports_key='outputs'):
```

Comment:
is `output` a port key, or a type of port? I keep thinking of the next level down as the port keys -- the specific names of the port, which is an output port.

Reply:
This is correct, so we need a new name for these. In a way, if 'inputs' is a port, then `inputs` is a port-key.... open to all ideas here. 

Perhaps this is a `ports_key`, which is a key to all of one type of ports (inputs or outputs), then it would be fine. So all the subkeys would be `port_keys` united by the overall `ports_key`, but I feel that distinction is too subtle and potentially confusing. 

`outputs` is a *set* of ports. A few options for the name of what is both a set of `inputs` and `outputs`:
* transport (maybe too much like biological transport)
* interface
* surface
* exchange
* missile
* .... ? 

Many other options. Let's brainstorm.